pkg/provider/azure/azclients/azclients.go

```diff
@@ -0,0 +1,97 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
```

Comment:
These license tags aren't needed

Reply:
could you explain why they are not needed?

Reply:
We don't need the license tag if they share the license defined in https://github.com/microsoft/retina/blob/main/LICENSE